lib/perl/Genome/Model/ClinSeq/Command/CreateMutationSpectrum.pm

```diff
@@ -251,6 +251,13 @@ sub get_mutation_spectrum_sequence_context_result {
   my $final_name = shift;
   my $tier1_snvs = shift;
   my $sub_outdir2 = shift;
+
+  #Check for tier1 variants
+  if(-z $tier1_snvs) {
+      $self->status_message("Tier1 SNVs file empty. Skipping mutation spectrum" .
+                            "sequence context for tier1.");
+      return;
+  }
```

Comment:
Maybe it would be better to switch the test.  That way if more code is added to this method in the future, it'll still get run.
```
if (-s $tier1_snvs) {
    ... # existing code
} else {
    $self->status_message('Skipping mutation spectrum')
}
```